src/NugetReferenceResolver/RuntimeGraph.cs

```diff
@@ -0,0 +1,926 @@
+﻿using Microsoft.DotNet.PlatformAbstractions;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace NugetReferenceResolver
+{
+    public class RuntimeGraph
+    {
+        private const string RuntimeCompatibility = @"
```

Comment:
Yep, but that is only available on .net core and this thing is cross-compiled, so just put it in here and skipped the dependency. Its basically reference data.